testing/go/types_test.go

```diff
@@ -117,6 +117,7 @@ var typesTests = []ScriptTest{
 		},
 	},
 	{
+		Skip: true,
```

Comment:
Maybe this is failing due to one of those subtle logic changes, so perhaps we should try this again after those changes have been made.